.gitignore

```diff
@@ -139,5 +139,5 @@ dmypy.json
 
 # my stuff 
 *.sub
-*.root
+
```

Comment:
I removed the *.root line because it's duplicate (there is the same line at the top of the file)